src/OpenTelemetry.Contrib.Instrumentation.AWSLambda/Implementation/AWSLambdaWrapper.cs

```diff
@@ -89,80 +89,80 @@ public class AWSLambdaWrapper
         }
 
         /// <summary>
-        /// Tracing wrapper for async Lambda handler.
+        /// Tracing wrapper for Lambda handler.
```

Comment:
Please look at the commits in the diff separately, I had to move a function around and that causes to the diffing algorithm to go off the rails apparently.